modules/nw-aws-nlb-existing-cluster.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic--aws-network-load-balancer.adoc
+
+[id="nw-aws-nlb-existing-cluster_{context}"]
+= Configuring an Ingress Controller Network Load Balancer on an existing AWS cluster
+
+You can create an ingress controller backed by an AWS Network Load Balancer (NLB) on an existing cluster.
+
+.Prerequisites
+
+* You must have an installed AWS cluster with network customizations.
```

Comment:
What is meant by "with network customizations"? My understanding is that NLB should be supported on standard AWS clusters.

Reply:
Now I think I understand why you're referencing "network customizations". [Network customizations](https://docs.openshift.com/container-platform/4.5/installing/installing_aws/installing-aws-network-customizations.html#installation-initializing_installing-aws-network-customizations) are only needed for __installing__ a cluster with NLB (Classic LB is the default).

Reply:
Sorry for the delay on this. I can't actually include the link to installation guide in the module, so I was referring to the title.